.buildkite/scripts/cloud-cleanup.sh

```diff
@@ -168,8 +169,9 @@ jq -c '.Clusters[]' redshift_clusters.json | while read i ; do
         #   --skip-final-cluster-snapshot
         echo "Done."
     fi
-done
+done <<< "$(jq -c '.Clusters[]' redshift_clusters.json)"
```

Comment:
This change is to ensure that the while command is not run in a subshell as it happens running after a pipe.

If `while` runs in a subshell, none of the variables modified in those iterations would be remain their values after finishing the loop.

Reply:
Would it work with a for loop like `for i in $(jq -c '.Clusters[]' redshift_clusters.json)`?

Reply:
Tested here: https://github.com/elastic/elastic-package/pull/1848/commits/0e34e0bc4b7d56cb3e0d963d04253f9b50492aaf
but it fails: https://buildkite.com/elastic/elastic-package-cloud-cleanup/builds/85#018f9aad-05c8-4703-b1d2-9c3c777cccf0/112-114

I'll keep the while loop